### ISSUE TITLE
Perks for increasing the squad do not work. Moreover, ALL the perks associated with increasing the squad do not work.

### ISSUE THREAD

**Issue by Vova1408** [Tyler-IN/MnB2-Bannerlord-CommunityPatch#151]

**Comment by Skau** [comment]
Look at #139. Maybe it helps?

**Comment by Skau** [comment]
TLDR; If you are using Bannerlord Tweaks with PartySizeTweak enabled, it's probably overriding the perks we have fixed that affect party size.

**Comment by Vova1408** [comment]
> TLDR; если вы используете настройки Bannerlord с включенным PartySizeTweak, это, вероятно, переопределяет льготы, которые мы исправили, которые влияют на размер партии.

Thank you for your answer. But the problem is that I don't use the Bannerlord Tweaks mod. As I understand from dialog number 139, the problem is that The bannerlord Tweaks mod and Community Patch are together in the same build. But I only have a Community Patch and still the perks for increasing the squad do not work.. Or did I misunderstand the dialogue?

**Comment by Vova1408** [comment]
> TLDR; If you are using Bannerlord Tweaks with PartySizeTweak enabled, it's probably overriding the perks we have fixed that affect party size.

Thank you for your answer. But the problem is that I don't use the Bannerlord Tweaks mod. As I understand from dialog number 139, the problem is that The bannerlord Tweaks mod and Community Patch are together in the same build. But I only have a Community Patch and still the perks for increasing the squad do not work.. Or did I misunderstand the dialogue?

**Comment by Skau** [comment]
What version of the game are you on?
Bannerlord Tweaks and Community Patch are two separate mods, not packed together in the same build. 
What perks are you having issues with specifically?

**Comment by Vova1408** [comment]
> What version of the game are you on?
> Bannerlord Tweaks and Community Patch are two separate mods, not packed together in the same build. There are some compatibility issues between them at the moment that we are looking into fixing.
> What perks are you having issues with specifically?

With the squire perk, as well as in all combat skills, where perks increase the size of the squad. I'm playing beta version 1.1.0

**Comment by Skau** [comment]
Squires is not yet on a release. You would have to compile from source code to get it. I don't believe we have fixed any of the combat related party size perks, only Steward and Squires from Riding. If you have the time list the ones that are not working, we can create issues on them and fix them :)

**Comment by Vova1408** [comment]
> На какой версии игры вы находитесь?
> Bannerlord Tweaks и Community Patch-это два отдельных мода, не упакованных вместе в одну сборку. Есть некоторые проблемы совместимости между ними на данный момент, что мы рассматриваем в фиксации.
> С какими конкретно льготами у вас возникают проблемы?
I meant that I don't have a Bannerlord Tweaks mod, so there shouldn't be any problems. And yet, I still face this problem.

**Comment by Skau** [comment]
We haven't fixed all party related perks yet. If you look at the main page on nexus mods [LINK](https://www.nexusmods.com/mountandblade2bannerlord/mods/186) you can see the perks we have fixed that has been released. If the perk is not mentioned, then we haven't fixed it yet.

**Comment by Vova1408** [comment]
> Оруженосцы еще не вышли на свободу. Вы должны были бы скомпилировать из исходного кода, чтобы получить его. Я не думаю, что мы исправили какие-либо из связанных с боем партийных льгот, только стюард и оруженосцы от верховой езды. Если у вас есть список времени те, которые не работают, мы можем создать проблемы на них и исправить их :)



> Squires is not yet on a release. You would have to compile from source code to get it. I don't believe we have fixed any of the combat related party size perks, only Steward and Squires from Riding. If you have the time list the ones that are not working, we can create issues on them and fix them :)

I'm sorry, Sir, I don't speak English very well, and I don't quite understand. You fixed the Squire's perk from riding and it should work? Or have you not fixed it yet?

**Comment by Vova1408** [comment]
> We haven't fixed all party related perks yet. If you look at the main page on nexus mods [LINK](https://www.nexusmods.com/mountandblade2bannerlord/mods/186) you can see the perks we have fixed that has been released. If the perk is not mentioned, then we haven't fixed it yet.

That's it, now I understand you. Thank you very much for your clarification!

**Comment by Skau** [comment]
No problem. It's fixed internally, you cannot download a version including it anywhere yet. We need to release a new version of the mod first. You will have to compile the source code yourself if you want the Squire fix :)

**Comment by Skau** [comment]
We should release a new version very soon though!

**Comment by Vova1408** [comment]
> Мы должны выпустить новую версию очень скоро, хотя!



> We should release a new version very soon though!

This is going to be great! You are the best, guys!
